src/routes/Course.tsx

```diff
@@ -88,20 +89,10 @@ const Course = () => {
           )}
         </div>
       </div>
-      <Title size="h2">Alle Kurse</Title>
-      {filteredCourses.length === 0 ? (
-        <Empty
-          style={{ marginBottom: '64px' }}
-          description={
-            'Es gibt im Moment keine Kurse' +
-            (userContext.user.type === 'pupil' ? ' für deine Klassenstufe' : '')
-          }
-        ></Empty>
-      ) : (
-        filteredCourses.map((c) => {
-          return <MyCourseCard course={c} />;
-        })
```

Comment:
I guess the filter to only show "fitting" courses in the user section isn't implemented yet.